vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,562 +1,225 @@
-# Business Objective
+---
+sidebarDepth: 2
+---
 
-The Entando Process Driven Applications plugin is built to provide a
-rich and full featured user experience to allow for the management and
-completion of business processes and automation. The UX is delivered a
-general purpose UX layer built using micro frontends that can be
-utilized for any business process or task engine. Customers drive the UX
-layer by utilizing a pre-developed Entando integration or by
-implementing a set of interfaces on the server side. The backend is a
-Spring Boot microservice that provides a pluggable interface allowing
-for the injection of any underlying process or automation toolkit. The
-interfaces and steps for creating a new PDA backend implementation are
-provided below. This document provides an overview of the Entando
-Process Driven Apps (PDA) plugin. The initial section covers the
-business functionality and user facing micro frontends. The second
-section covers the architecture and the points of extension for the PDA
-plugin. The third provides a detailed technical overview for developers
-to add new functionality to a PDA implementation.
+# Entando Process Driven Applications Plugin
 
-# Terminology
+## Introduction
 
-## Micro Frontend
+The Entando Process Driven Applications (PDA) plugin is engineered to provide a rich and full-featured user experience while facilitating the management and completion of business processes and automation.  This solution template includes: 
 
-A Micro frontend architecture is an approach to developing web
-application as a composition of small frontend apps. Instead of writing
-a large monolith frontend application, the application is broken down
-into domain specific micro frontends, which are self-contained and can
-be developed and deployed independently.
+- A general purpose UX layer created from micro frontends and can be implemented for any business process or task engine. The UX layer can serve data via the included Entando integration adapter or by implementing a set of interfaces on the server side.
+- A Spring Boot microservice backend providing a pluggable interface for the injection of underlying processes or automation toolkits. The interfaces and steps for creating a new PDA backend implementation are described in further detail [here](../customize-the-platform/pda-technical-guide.md).
 
-## Process Automation Manager (PAM)
+This tutorial will guide you through installing the PDA plugin using the Entando Component Repository (ECR) and an Entando bundle. The goal of this exercise is to demonstrate the scope of process automation functionality enabled by integrating the following components: 
```

Comment:
"This exercise demonstrates the scope of process automation enabled by integrating the following:"

since "Entando" directly precedes "bundle" should we capitalize? Entando Bundle? I feel like this is our established practice with Entando Application, etc.